src/sentry/integrations/github/integration.py

```diff
@@ -48,14 +49,17 @@
 }
 
 
-class GitHubIntegration(Integration, RepositoryMixin):
+class GitHubIntegration(Integration, GitHubIssueSync, RepositoryMixin):
 
     def get_client(self):
         return GitHubAppsClient(external_id=self.model.external_id)
 
     def get_repositories(self):
         return self.get_client().get_repositories()
 
+    def format_external_key(self, data):
+        return '{}#{}'.format(data.get('repo'), data.get('key'))
```

Comment:
i'd just do `data['repo']` and `data['key']` since it's probably better for that to raise a `KeyError` than to return a weird incomplete issue key

Reply:
also, since `data` should have a standardized format since you're just passing in the result of `get_issue` or `create_issue` i think it would probably just be a good idea to add this to the base `IssueSyncMixin` class as
```python
def format_external_key(self, data):
    """
    Takes result of `get_issue` or `create_issue` and returns the formatted key
   """
   return data['key']
```
that _should_ work for jira and the example integration. cc @lauryndbrown to confirm if that will work for vsts